gen/nl/hannahsten/texifyidea/grammar/LatexLexer.java

```diff
@@ -53,13 +54,13 @@
   /**
    * ZZ_LEXSTATE[l] is the state in the DFA for the lexical state l
    * ZZ_LEXSTATE[l+1] is the state in the DFA for the lexical state l
-   *                  at the beginning of a line
```

Comment:
What version of jflex do you have? When I run 1.7.0-2 it reverts these changes for me.
It's very unfortunate that we couldn't integrate generating these classes in the Gradle workflow, but I haven't heard of any progress that would enable that so we have to keep them committed for now.

Reply:
I also have jflex-1.7.0-2. I am not sure whether I understand your comment though. Do you mean the changes introduced by the PR?

Reply:
Did you edit LatexLexer.java by hand? Because you don't have to: ctrl+shift+g (or right-click) in latexlexer.flex generates the LatexLexer.java, that's why it's in /gen

Reply:
No, I didn't (at least not on purpose). I generated the lexer just like you describe. I really don't know how these changes happened. Anyway, I let the generator run again (4cdf503) and it changed LatexLexer.java, hopefully to the same state you receive when running the generator.